server.py

```diff
@@ -72,6 +88,25 @@ def parse_request(request):
         return meth[1]
 
 
+def resolve_uri(parse):
+    root = os.path.join(os.getcwd(), 'webroot')
+    body = ''
+    content_type = ''
+    if os.path.isdir(root + parse):
```

Comment:
Better to use `os.path.join` to build filesystem paths than to use string concatenation.